sertmer/swapi-trivia
# Sign out

#### What’s this PR do?
Allows for the sign out button to reset State

#### Where should the reviewer start?
In App to view the method, and then Header to see where it's passed.

#### How should this be manually tested?
Fill out input fields, then click sign out to see appended information be reset.
